### PR TITLE
resource/alicloud_cloud_firewall_control_policy_order: Improved alicloud_cloud_firewall_control_policy_order testcase

### PR DESCRIPTION
**Pull request by MrWolong**

### CODE REVIEW

File: alicloud/resource_alicloud_cloud_firewall_control_policy_order.go [hunk 1]
```diff
@@ -32,17 +30,17 @@ func resourceAlicloudCloudFirewallControlPolicyOrder() *schema.Resource {
 				Type:         schema.TypeString,
 				Required:     true,
 				ForceNew:     true,
-				ValidateFunc: validation.StringInSlice([]string{"in", "out"}, false),
+				ValidateFunc: StringInSlice([]string{"in", "out"}, false),
 			},
 			"order": {
```
Comment (MrWolong) [inline alicloud/resource_alicloud_cloud_firewall_control_policy_order.go:35]: order实际是必填的，不传openapi会报错

File: website/docs/r/cloud_firewall_control_policy_order.html.markdown [hunk 2]
```diff
@@ -2,58 +2,60 @@
...
 
 The following arguments are supported:
 
-
-* `acl_uuid` - (Required) The unique ID of the access control policy.
-* `direction` - (Required) Direction. Valid values: `in`, `out`.
-* `order` - (Optional) The priority of the access control policy. The priority value starts from 1. A small priority value indicates a high priority. **NOTE:** The value of -1 indicates the lowest priority.
+* `acl_uuid` - (Required, ForceNew) The unique ID of the access control policy.
+* `direction` - (Required, ForceNew) The direction of the traffic to which the access control policy applies. Valid values: `in`, `out`.
+* `order` - (Required, Int) The priority of the access control policy. The priority value starts from 1. A small priority value indicates a high priority. **NOTE:** The value of `-1` indicates the lowest priority.
```
[the remark this answers is not in the document]
Comment (MrWolong) [reply website/docs/r/cloud_firewall_control_policy_order.html.markdown:52]: 已修复

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Pull request merged**